.gitignore

```diff
@@ -28,7 +28,6 @@ yarn-error.log*
 generated-cra-app
 .cra-template-rb
 template
-.env
```

Comment:
keep .env in gitignore, vercel is used for handling that.

Reply:
Okay thank you, I will practice it.